### PR TITLE
Add signal-desktop-bin ebuild

### PR DESCRIPTION
**Pull request by robert7k**
Bug: https://bugs.gentoo.org/636164

### CODE REVIEW

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+IUSE=""
+
+RESTRICT="bindist mirror"
+
+DEPEND="
+	gnome-base/gconf:2
+	dev-libs/libappindicator:3
+	dev-libs/nss
+	x11-libs/libXtst
+	"
```
Comment (SoapGentoo) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: if this is a binary package, you don't need `DEPEND`

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+RDEPEND="${DEPEND}"
+
+QA_PREBUILT="opt/Signal/signal-desktop
+	opt/Signal/libnode.so
+	opt/Signal/libffmpeg.so"
+
+S="${WORKDIR}"
+
+src_install() {
+	doins -r "${S}/"*
```
Comment (SoapGentoo) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: `doins -r .`

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	fperms +x /opt/Signal/signal-desktop
+
+	dosym /opt/Signal/${MY_PN} usr/bin/${MY_PN}
```
Comment (SoapGentoo) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: `dosym ../../opt/Signal/${MY_PN} /usr/bin/${MY_PN}` (use relative symlinks)

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit eutils gnome2-utils unpacker xdg-utils
```
Comment (SoapGentoo) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: for what do you need `eutils`?

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild (hunk 1 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: Changed.

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild (hunk 2 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: Done

File: net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild (hunk 3 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild]: Done

**Pull request closed without merging**